spec/models/case_attachment_upload_group_spec.rb

```diff
@@ -34,8 +28,22 @@
   end
 
   describe "date_time" do
-    it "returns the formatted upload group time converted to BST" do
-      expect(upload_group.date_time).to eq "08 Jun 2017 11:11"
+    context "when timezone offset is BST" do
+      let(:timestamp) { "20170608111112" }
+      let(:upload_group) { described_class.new([timestamp, @responder.id], :responder, @kase, @kase.attachments) }
+
+      it "returns the formatted upload group time in BST" do
+        expect(upload_group.date_time).to eq "08 Jun 2017 11:11"
+      end
+    end
+
+    context "when timezone offset is GMT" do
+      let(:timestamp) { "20170608101112" }
+      let(:upload_group) { described_class.new([timestamp, @responder.id], :responder, @kase, @kase.attachments) }
+
+      it "returns the formatted upload group time in GMT" do
+        expect(upload_group.date_time).to eq "08 Jun 2017 10:11"
```

Comment:
This doesn't matter anymore because of my previous comment, but if the date was 8th June, you would expect a BST time to be returned.